packages/plugin/README.md

```diff
@@ -1,6 +1,16 @@
-# Introduction
+<div align='center'>
 
-## Theia Plugin system description
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>THEIA - PLUGIN EXTENSION</h2>
+
+<hr />
+
+</div>
+
+## Description
 
```

Comment:
Would be nice to put some description

Reply:
@lmcbout I thought the description would be broken down into the content that was already present, please see the rendered version:
https://github.com/eclipse-theia/theia/blob/57cf060e2dd43370dd5ac5b27c828d17448ac553/packages/plugin/README.md